### PR TITLE
Adding anaerobic digester cost

### PR DESCRIPTION
**Pull request by agarciadiego**
## Fixes/Resolves:

Addresses #1360 for Anaerobic Digester

## Summary/Motivation:

Required for WaterTap 1.0

## Changes proposed in this PR:
- Adds documentation for anaerobic digester
-

### Legal Acknowledgement

By contributing to this software project, I agree to the following terms and conditions for my contribution:

1. I agree my contributions are submitted under the license terms described in the LICENSE.txt file at the top level of this directory.
2. I represent I am authorized to make the contributions and grant the license. If my employer has rights to intellectual property that includes these contributions, I represent that I have received permission to make contributions and grant the required license on behalf of that employer.

### CODE REVIEW

File: docs/technical_reference/costing/anaerobic_digester.rst [hunk 1]
```diff
@@ -9,45 +9,34 @@ The following parameters are constructed for the unit on the FlowsheetCostingBlo
 .. csv-table::
    :header: "Description", "Symbol", "Parameter Name", "Default Value", "Units"
 
-   "description", ":math:`Symbol_{example}`", "``parameter_name``", "1", ":math:`\text{dimensionless}`"
+   "Capital cost A parameter :math:`^1`", ":math:`A`", "``capital_a_parameter``", "19.3552312e6", ":math:`\text{USD}_{2012}`"
+   "Capital cost B parameter :math:`^1`", ":math:`B`", "``capital_b_parameter``", "0.6", ":math:`\text{dimensionless}`"
+   "Reference flow :math:`^1`", ":math:`F_r`", "``911.054``", "0.6", ":math:`m^3/h`"
```
Comment (MarcusHolly) [inline docs/technical_reference/costing/anaerobic_digester.rst]:
```suggestion
   "Reference flow :math:`^1`", ":math:`F_r`", "``reference_flow``", "911.054", ":math:`m^3/h`"
```

File: docs/technical_reference/costing/anaerobic_digester.rst [hunk 2]
```diff
@@ -9,45 +9,34 @@ The following parameters are constructed for the unit on the FlowsheetCostingBlo
...
 Costing Method Variables
 ++++++++++++++++++++++++
 
-The following variables are constructed on the unit block (e.g., m.fs.unit.costing) when applying the `cost_anaerobic_digester` costing method in the ``watertap_costing_package``:
-
-.. csv-table::
-   :header: "Description", "Symbol", "Variable Name", "Index", "Units"
-
-   "description", ":math:`Symbol_{example}`", "``variable_name``", "[t]", ":math:`\text{dimensionless}`"
+There are no costing method variables unique to the CSTR with injection.
```
Comment (MarcusHolly) [inline docs/technical_reference/costing/anaerobic_digester.rst]:
```suggestion
There are no costing method variables unique to the anaerobic digester.
```

File: docs/technical_reference/costing/anaerobic_digester.rst [hunk 3]
```diff
@@ -9,45 +9,34 @@ The following parameters are constructed for the unit on the FlowsheetCostingBlo
...
-
-    .. math::
-
-        C_{cap,tot} = C_{cap,example1}+C_{cap,example2}+C_{cap,other}
+Capital cost is dependent upon the unit's volumetric flowrate, :math:`F`, as shown in the equation below.
 
     .. math::
 
-        C_{cap,example1} = fill in equation for each component in total capex equation
+        C_{cap,tot} = A * F/F_r*{B}
```
Comment (MarcusHolly) [inline docs/technical_reference/costing/anaerobic_digester.rst]:
```suggestion
        C_{cap,tot} = A * F/F_r^{B}
```
I think B should be an exponent here

File: docs/technical_reference/costing/anaerobic_digester.rst [hunk 4]
```diff
@@ -9,45 +9,34 @@ The following parameters are constructed for the unit on the FlowsheetCostingBlo
...
 
  
 Operating Cost Calculations
 +++++++++++++++++++++++++++
 
-Describe operating/maintenance costs..keep it concise where possible
+Electricity :math:`C_{elec}` is a variable operating cost based on the energy intensity :math:`E` of the unit process
+(electricity consumption for the CSTR with injection), electricity price :math:`P`, electricity flow :math:`Q`, and the plant
```
Comment (MarcusHolly) [inline docs/technical_reference/costing/anaerobic_digester.rst]:
```suggestion
(electricity consumption for the anaerobic digester), electricity price :math:`P`, electricity flow :math:`Q`, and the plant
```

**Review by MarcusHolly — COMMENTED** [review]
A few typos

**Review by MarcusHolly — APPROVED** [review]

File: docs/technical_reference/costing/anaerobic_digester.rst [hunk 5]
```diff
@@ -9,45 +9,34 @@ The following parameters are constructed for the unit on the FlowsheetCostingBlo
...
-
-    .. math::
-
-        C_{cap,tot} = C_{cap,example1}+C_{cap,example2}+C_{cap,other}
+Capital cost is dependent upon the unit's volumetric flowrate, :math:`F`, as shown in the equation below.
 
     .. math::
 
-        C_{cap,example1} = fill in equation for each component in total capex equation
+        C_{cap,tot} = A * F/F_r^{B}
```
Comment (luohezhiming) [inline docs/technical_reference/costing/anaerobic_digester.rst]:
```suggestion
        C_{cap,tot} = A * (F/F_r)^{B}
```

Comment (luohezhiming) [reply docs/technical_reference/costing/anaerobic_digester.rst]: I think add the parentheses is helpful to avoid misunderstanding

**Review by luohezhiming — APPROVED** [review]

**Pull request merged**